src/search/landmarks/landmark_factory_merged.cc

```diff
@@ -142,13 +139,9 @@ void LandmarkFactoryMerged::generate(const TaskProxy &task_proxy) {
        necessary to do so again here, so these steps are omitted. For
        reasonable orders, acyclicity of the landmark graph and the costs of
        landmarks we should also determine this.
+
+       Do we still need this comment?
```

Comment:
It is not true what the comment says, right? I mean, it is only true if all the individual landmark factories are set up accordingly. However, it is perfectly fine to do something like `lm_merged([lm_rhw(disjunctive_landmarks=true), lm_hm(disjunctive_landmarks=false)], disjunctive_landmarks=false)`. I know the example is outdated because `lm_hm` does not produce disjunctive landmarks and doesn't have the orderings anymore. What I wanted to say is that without doing the postprocessing for disjunctive landmarks in `lm_merged`, the outcome will have disjunctive landmarks even though I've specifically set the option that it shouldn't have.

Reply:
I think we discussed this topic when discussing the design of having factories to post process landmark graphs, like the one for making it acyclic, right? Anyway, the TODO comment indeed seems to be wrong and should go away.